spec/requests/hosts_spec.rb

```diff
@@ -94,8 +45,8 @@
         params = {
           "action"    => "edit",
           "resources" => [
-            {:id => host1.id, :authentications => {"default" => {:username => "abc", :password => "abc123"}}},
-            {:id => host2.id, :authentications => {"default" => {:username => "def", :password => "def456"}}}
+            {:id => host1.id, "authentications" => {"default" => {"username" => "abc", "password" => "abc123"}}},
+            {:id => host2.id, "authentications" => {"default" => {"username" => "def", "password" => "def456"}}}
```

Comment:
I think id should be a string here too, right?
```suggestion
            {"id" => host1.id, "authentications" => {"default" => {"username" => "abc", "password" => "abc123"}}},
            {"id" => host2.id, "authentications" => {"default" => {"username" => "def", "password" => "def456"}}}
```

Reply:
Done